task1/src/Counter.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+/*
+Counter Using React.Component
+ */
+export class Counter extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            counter: 0
+        }
+    }
+
+    changeCounter = (increment) => {
+        if (increment) {
+            this.setState(prevState => ({ counter: prevState.counter + 1}));
+        } else {
+            this.setState(prevState => ({ counter: prevState.counter - 1}));
+        }
+    }
+
+    render() {
+        return <div>
```

Comment:
good practice to add () brackets and start jsx from new line 

or better as arrow function: 
```
render = () => (
  <div> 
    ...
  </div>
)
```

Reply:
updated.